labelCloud/model/point_cloud.py

```diff
@@ -43,10 +42,10 @@ class PointCloud(object):
     def __init__(
         self,
         path: Path,
-        points: np.ndarray,
+        points: npt.NDArray[np.float32],
+        label_definition: Dict[str, int],
```

Comment:
I would rather see the definition / schema as a global variable, maybe accessible via a cached function or similar.

But this is probably then part of the pending PR, so would only need to be adapted then.

Reply:
agree - now the label definition is used all over the place it makes sense it's a skeleton. I'll keep it as is for now.